zone/exp.cpp

```diff
@@ -1011,7 +1011,7 @@ bool Group::ProcessGroupSplit(Mob* killed_mob, struct GroupExpSplit_Struct& gs,
 			{
 				Client *cmember = members[x]->CastToClient();
 				if (cmember->CastToClient()->GetZoneID() == zone->GetZoneID() && 
-					cmember->IsInLevelRange(maxlevel) && 
+					cmember->IsInLevelRange(maxlevel) &&
```

Comment:
Just a misc note, since this line was only checking about green con stuff, it's the only place I explicit didn't use the `(level, level2)` logic on purpose. But in general, all exp level range checks should now always pass `(level, level2)` to for the correct logic for SF and non-SF players to go through one code path.